### PR TITLE
Milestone: Complete project

### PR DESCRIPTION
**Pull request by kelvinbush**
In this pull request I have completed the project here is what I did:
- Set up React project with CRA template.
- Set up skeletons for home and detail page.
- Added styling with `sass`.
- Connected to Coin-Gecko API to fetch crypto currency metrics.
- Added Redux for complex state.
- Added unit and integration tests.
- Set up linters for code quality.
- Added documentation to project.
- Deployed to Netlify [link](https://block-cypher.netlify.app/)

The video presentation is available [here](https://www.loom.com/share/4236f7074cc049068789d6eb6763a472)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/14ab8586-1e4c-4dcf-9627-c5d106a9fd32/deploy-status)](https://app.netlify.com/sites/block-cypher/deploys)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto-tracker
 
-## Available Scripts
+ This is a React app consuming the [Coin Gecko](https://www.coingecko.com/en/api/documentation).
+ Displaying the metrics and statistics of top cryptocurrencies.
```
Comment (Deepakdanger) [inline README.md:7]: - [ ] Kindly explain a little more about the description of the project.
- [ ] Kindly add some project screenshots here.

File: src/components/ChangeIndicator/ChangeIndicator.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
```
Comment (Deepakdanger) [inline src/components/ChangeIndicator/ChangeIndicator.js]: - [ ] kindly remove this disabled Eslint and solve it manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @kelvinbush, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] At many files the linter is disabled, Kindly remove all those & resolve it manually.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @kelvinbush,

✔️ Good documentation
✔️ Nice design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**